PerimeterXModule/Internals/Validators/PXCookieValidator.cs

```diff
@@ -55,38 +91,43 @@ public bool CookieVerify(PxContext context, IPxCookie pxCookie)
 				if (string.IsNullOrEmpty(pxCookie.Hmac))
 				{
 					Debug.WriteLine("Request with invalid cookie (missing signature) - " + context.Uri, PxConstants.LOG_CATEGORY);
-					context.S2SCallReason = RiskRequestReasonEnum.INVALID_COOKIE;
+					context.S2SCallReason = CALL_REASON_VALIDATION_FAILED;
```

Comment:
Why not `CALL_REASON_INVALID_COOKIE`?

Reply:
there are no detections on `CALL_REASON_INVALID_COOKIE`, nothing returns `CALL_REASON_INVALID_COOKIE`, its not part of the detectors thus this will make it confusing, we can either do new design
`CALL_REASON_INVALID_COOKIE` is something that was coded in the begning but its not part of the original enforcer today

Reply:
👍